spec/support/precompile_assets.rb

```diff
@@ -1,23 +1,23 @@
-RSpec.configure do |config|
-  config.before(:suite) do
-    examples = RSpec.world.filtered_examples.values.flatten
-    has_no_system_tests = examples.none? { |example| example.metadata[:type] == :system }
+# RSpec.configure do |config|
+#   config.before(:suite) do
+#     examples = RSpec.world.filtered_examples.values.flatten
+#     has_no_system_tests = examples.none? { |example| example.metadata[:type] == :system }
 
-    if has_no_system_tests
-      $stdout.puts "\n🚀️️  No system test selected. Skip assets compilation.\n"
-      next
-    end
+#     if has_no_system_tests
+#       $stdout.puts "\n🚀️️  No system test selected. Skip assets compilation.\n"
+#       next
+#     end
 
-    $stdout.puts "\n🐢  Precompiling assets.\n"
-    original_stdout = $stdout.clone
+#     $stdout.puts "\n🐢  Precompiling assets.\n"
+#     original_stdout = $stdout.clone
 
-    start = Time.current
-    begin
-      $stdout.reopen(File.new('/dev/null', 'w'))
-      system('bin/rails assets:precompile')
-    ensure
-      $stdout.reopen(original_stdout)
-      $stdout.puts "Finished in #{(Time.current - start).round(2)} seconds"
-    end
-  end
-end
+#     start = Time.current
+#     begin
+#       $stdout.reopen(File.new('/dev/null', 'w'))
+#       system('bin/rails assets:precompile')
+#     ensure
+#       $stdout.reopen(original_stdout)
+#       $stdout.puts "Finished in #{(Time.current - start).round(2)} seconds"
+#     end
+#   end
+# end
```

Comment:
Can this file be deleted?

Reply:
Good spot, thanks Josh. I hoped we could get rid of it, but think we might still need it. I ran into an issue locally where I had done something to the assets on another branch, switched to this branch, then ran the tests and started getting failures because it was using asset builds from the other branch 😵‍💫